sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,152 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_resetWavefrontCorrection</EFDB_Topic>
+    <EFDB_Topic>MTAOS_command_resetCorrection</EFDB_Topic>
     <Description>Reset the current wavefront error correction. The aggregated degree of freedom in optical feedback control will be reset to the initial state.</Description>
-    <item>
-      <EFDB_Name>value</EFDB_Name>
-      <Description>This value is ignored.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_issueWavefrontCorrection</EFDB_Topic>
-    <Description>Issue the wavefront correction to subsystems based on the most recently measured wavefront error.</Description>
-    <item>
-      <EFDB_Name>value</EFDB_Name>
-      <Description>This value is ignored.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTAOS_command_issueCorrection</EFDB_Topic>
+    <Description>Issue the wavefront correction to subsystems based on the most recent OFC outputs.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processCalibrationProducts</EFDB_Topic>
-    <Description>Process new calibration products.</Description>
-    <item>
-      <EFDB_Name>directoryPath</EFDB_Name>
-      <Description>Absolute directory path to calibration products.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTAOS_command_rejectCorrection</EFDB_Topic>
+    <Description>Reject the most recent wavefront correction.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processWavefrontError</EFDB_Topic>
```

Comment:
Is that the ISR only?

Reply:
Uhm... this comment seems to have been made in a command that was removed. Are you asking about the `processWavefrontError` or the `selectSources` command?